targets/XPlatCppSdk/source/code/include/playfab/PlayFabTransportHeaders.h

```diff
@@ -1,12 +1,24 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// 
+// This header file is used to include headers of transport plugins supported on each platform.
+
 #pragma once
 
 #include <playfab/PlayFabPlatformMacros.h>
 
 #ifdef PLAYFAB_PLATFORM_XBOX
 #include <playfab/PlayFabIXHR2HttpPlugin.h>
+#include <playfab/OneDSIXHR2HttpPlugin.h>
 #endif // PLAYFAB_PLATFORM_XBOX
 
-#if defined(PLAYFAB_PLATFORM_WINDOWS) || defined(PLAYFAB_PLATFORM_LINUX) 
-#include <curl/curl.h>
+#ifdef PLAYFAB_PLATFORM_WINDOWS
+#include <playfab/PlayFabWinHttpPlugin.h>
+#include <playfab/PlayFabHttp.h>
+#include <playfab/OneDSWinHttpPlugin.h>
+#include <playfab/OneDSHttpPlugin.h>
```

Comment:
>OneDSHttpPlugin [](start = 18, length = 15)

remove ?

Reply:
We are not removing anything for Windows as part of this (it may also break many things at once). There will be a separate task/story for it. 

---
In reply to: [244867295](https://github.com/PlayFab/SDKGenerator/pull/368#discussion_r244867295) [](ancestors = 244867295)